core/include/usServiceProperties.h

```diff
@@ -43,97 +41,6 @@ namespace us {
  */
 typedef std::unordered_map<std::string, Any> ServiceProperties;
```

Comment:
In issue #85 using a `Properties` typedef is mentioned. So renaming the above to `Properties` and using it consistently everywhere is what I thought #85 is about, right?